core/handlers/v2/simulation_views_v5.go

```diff
@@ -40,30 +40,43 @@ type MatcherViewV5 struct {
 	Config  map[string]interface{} `json:"config,omitempty"`
 }
 
+type LiteralViewV5 struct {
+	Name  string      `json:"name"`
+	Value interface{} `json:"value"`
+}
+
+type VariableViewV5 struct {
+	Name       string `json:"name"`
+	Method     string `json:"method"`
+	Expression string `json:"expression"`
```

Comment:
How about calling this `Function` and `Arguments`/`Args` instead? Because it could be used to declare any templating functions with arguments, not only for the xpath or jsonpath functions

Reply:
We can have func and args globally as it will be much easier.

Reply:
@tommysitu  I have made the required changes. If the changes look good, then I can add more UTs and documentation.

Reply:
@tommysitu  You can review completely. Tested and added documentation for the same.